usr/lib/sticky/common.py

```diff
@@ -41,17 +41,25 @@ def __init__(self, settings, window):
         self.notes_lists = {}
 
         if os.path.exists(CONFIG_PATH):
-            self.load_notes()
+            self.load_notes(window)
```

Comment:
We don't actually need to pass window as an argument because self already has window set on it. Instead, you can just pass self.window to the message dialog down below.

Reply:
resolved